### PR TITLE
calibration

### PR DESCRIPTION
**Pull request by cfinn68916**
This adds a place for users to calibrate their camera. I'm still working on a more formal page, but this works as a start.

### CODE REVIEW

File: SecondSight/Calibration.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+
+# mostly coppied from https://github.com/kyle-bersani/opencv-examples/blob/master/CalibrationByCharucoBoard/CalibrateCamera.py
```
Comment (joshbressers) [inline SecondSight/Calibration.py]: We need to treat this file different than the rest of the code due to being MIT licensed. We have to include the license notice from the original project

https://github.com/kyle-bersani/opencv-examples/blob/master/LICENSE

Keeping the URL there is also good
Copy and paste that license file into the top of this one

**Review by joshbressers — CHANGES_REQUESTED** [review]
I opened #25 to track adding a note about including the code of others. In this case MIT is a compatible license, but not all licenses are compatible.

We should also think about how to create functional tests for something like this in the future. It will be an interesting challenge.

File: SecondSight/Calibration.py (hunk 1 above)
Comment (cfinn68916) [reply SecondSight/Calibration.py]: To be clear, that comment is from the original file in DEATHSTARE++, and most of the code has been changed a lot.

Comment (joshbressers) [reply SecondSight/Calibration.py]: It's a very complicated topic actually. I'm happy to discuss it on Thursday. This is the safest option for sure as it's very likely the code is still covered under the MIT license

**Review by joshbressers — APPROVED** [review]

**Pull request merged**